README.md

```diff
@@ -20,29 +19,22 @@ Convene secure, affordable, always-available digital Spaces where anyone can
 
 ## Overview
 
-Convene serves as the _Means of Connection_. Create a Space to co-conspire,
-relax, learn, play, work, swap recipes, read books, or really anything!
+Convene is an _Operating System_ for **Community Owned Digital Infrastructure**. Once installed
+to the cloud, you access Convene via a web browser or on-device apps.
 
-Convene provides the underlying infrastructure for a _Neighborhood of Spaces_.
-_Operators_ help _Clients_ design their _Spaces_ and keep the Neighborhood up
-and running. _Maintainers_ and _Contributors_ enhance Convene with:
+Convene serves as a community-owned, more ethical alternative to investor-owned online tools like
+SquareSpace, Etsy, Patreon, Zoom, Yelp, and more.
 
-- _Furniture_, fun toys and powerful tools
-- _Utilities_, integrating external services like Stripe, Calendly, Jitsi, and
-  more.
+A Convene Deployment is a _Neighborhood_, which People can create _Spaces_ in, and fill those _Spaces_
```

Comment:
Remove capitals from "People", and maybe drop the "can", i.e:

> A Convene Deployment is a _Neighbordhood_, which people can create _Spaces_ in, and fill...

Also, potentially add a period in there to make the second part less awkward (without the dangling preposition), like:
> A Convene Deployment is _Neighborhood_. People create _Spaces_ in a _Neighborhood_, and fill those _Spaces_...

Reply:
fixed in https://github.com/zinc-collective/convene/pull/749